rules_java_gapic/java_gapic_pkg.bzl

```diff
@@ -13,14 +13,25 @@
 # limitations under the License.
 
 load("@com_google_api_gax_java_properties//:dependencies.properties.bzl", "PROPERTIES")
+load("@com_google_protobuf//:protobuf_version.bzl", "PROTOBUF_JAVA_VERSION")
+
+def _syncVersionWithGoogleapis():
+    syncedProterties = {}
+
+    # Before this replacement, there is a problem (e.g., b/284292352) when
+    # the version of protobuf defined in googleapis is higher than protobuf
+    # defined in gax-java/dependencies.properties, use this replacement to
+    # sync the two versions.
+    syncedProterties["version.com_google_protobuf"] = PROTOBUF_JAVA_VERSION
+    return syncedProterties
 
 def _wrapPropertyNamesInBraces(properties):
     wrappedProperties = {}
     for k, v in properties.items():
         wrappedProperties["{{%s}}" % k] = v
     return wrappedProperties
 
-_PROPERTIES = _wrapPropertyNamesInBraces(PROPERTIES)
+_PROPERTIES = _wrapPropertyNamesInBraces(PROPERTIES | _syncVersionWithGoogleapis())
```

Comment:
```suggestion
# Before this replacement, there is a problem (e.g., b/284292352) when ...
PROPERTIES["version.com_google_protobuf"] = PROTOBUF_JAVA_VERSION
_PROPERTIES = _wrapPropertyNamesInBraces(PROPERTIES)
```

Reply:
This actually won't work as it will violate [mutability rule](https://docs.bazel.build/versions/0.24.0/skylark/language.html#mutability).
```
ERROR: Traceback (most recent call last):
        File "/private/var/tmp/_bazel_joewa/b6b64da05090260654456c80e641ce7c/external/gapic_generator_java/rules_java_gapic/java_gapic_pkg.bzl", line 28, column 43, in <toplevel>
                PROPERTIES["version.com_google_protobuf"] = PROTOBUF_JAVA_VERSION
Error: trying to mutate a frozen dict value
```

Reply:
Nice learning. Then create a copy.